Zohosign ms powerautomate change

ZohoSign-integ opened the pull request:
---
Please check the following conditions for your PR.

- [x] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [x] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

ptalrejagithub commented:
@ZohoSign-integ, can you please just submit the apiDefinationswagger.json, apiProperties.json and readme,md. The readme.md file is required for open sourcing your connector which can be helpful to the open-source users who may want to contribute in your connector. A reference of a readme.md file is [here](https://github.com/st0014/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md)

sriyen-msft commented:
@ZohoSign-integ Please remove your connector files from `custom-connectors` folder and update the PR

ZohoSign-integ reviewed (commented):
removed the zohosign files under custom connector and updated the readme file also.

ZohoSign-integ commented:
Hi Team,



I have corrected and updated the PR. Please review. 



https://github.com/microsoft/PowerPlatformConnectors/pull/843#pullrequestreview-669761314



Regards,

Gopi









---- On Thu, 27 May 2021 03:43:49 +0530 Srikanth Yenagandhula ***@***.***> wrote ----




https://github.com/ZohoSign-integ Please remove your connector files from custom-connectors folder and update the PR

—
You are receiving this because you were mentioned.
Reply to this email directly, https://github.com/microsoft/PowerPlatformConnectors/pull/843#issuecomment-849156367, or https://github.com/notifications/unsubscribe-auth/AT66YSB7RBGAMZ2WJWJXC53TPVXB3ANCNFSM45ROER4A.

sriyen-msft commented on certified-connectors/ZohoSign/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,1943 @@
+            "in": "formData",
+            "name": "email_id",
+            "type": "string",
+            "description": "[\"Enter the email ids of all the invitees whom you want to to be added\"]",
```
@ZohoSign-integ Can you please let us knw why this description is double quotes

ZohoSign-integ replied:
It is to highlight the sentence inside the square brackets, for better understanding.

ZohoSign-integ commented on certified-connectors/ZohoSign/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,1943 @@
+                        },
+                        "custom_domains": {
+                          "type": "boolean",
+                          "description": "custom_domains"
```
all the descriptions are corrected. Please review them @sriyen-msft
